bucket/redpen.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.10.1",
+    "description": "A proofreading tool.",
+    "homepage": "http://redpen.cc",
+    "license": "Apache-2.0",
+    "url": "https://github.com/redpen-cc/redpen/releases/download/redpen-1.10.1/redpen-1.10.1.tar.gz",
+    "hash": "39a148d3d89efef0e58ee7250e1bab7e26bf1edf83616934265c603623351fa0",
+    "extract_dir": "redpen-distribution-1.10.1",
+    "bin": [
+        "bin\\redpen.bat",
+        "bin\\redpen-server.bat"
+    ],
+    "suggest": {
+        "Java": [
+            "oraclejdk",
+            "some JRE from java bucket"
```

Comment:
I hoped you will look into Java bucket and find any appropriate JRE, which could be used

Reply:
```suggestion
            "adopt8-hotspot-jre"
```

Reply:
Thank you very much for everything! I applied your suggestion, and I checked RedPen worked with adopt8-hotspot-jre.